README.md

```diff
@@ -12,7 +12,7 @@ Example usage:
 > {-# LANGUAGE ScopedTypeVariables #-}
 > {-# LANGUAGE TemplateHaskell #-}
 > {-# LANGUAGE TypeApplications  #-}
-> {-# LANGUAGE TypeFamilies #-}
+> {-# LANGUAGE ConstraintKinds #-}
```

Comment:
I like to sort these so there's no ambiguity where someone should add a new one.